.github/ISSUE_TEMPLATE/bump_dependancies.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bump dependancies
+about: Bump vender dependencies for release
+title: 'Bump vender dependencies for release 0.0.0'
+labels: enhancement
```

Comment:
Was there not a talk about using a `dependency` label?

https://github.com/waku-org/pm/pull/91

Reply:
Yes indeed, looks like `dependencies` was settled on, I updated the template as such.